rfp-proposals/rfp-proposal-filscan.md

```diff
@@ -0,0 +1,141 @@
+ # RFP Proposal: Block Explorer
+ 
+ **Name of Project:** Filscan
+ 
+ **Link to RFP:**  *[rfp-block-explorer.md](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-block-explorer.md)*
+ 
+ **RFP Category:**  `app-dev`
+ 
+ **Proposer:**   *[IPFS-FORCE](https://github.com/orgs/ipfs-force-community)*
+ 
+ **Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:**  `Yes`
+ 
+ # Project Description
+ 
+Hi Labs, we are IPFS-Force team from China with several years' development experience in Blockchain Industry and deep understanding of Blockchain Activities and Actors. With solid experience, we are capable to dive and analyze multidimensional data from blockchain raw data and displayed in creative visualization.
+ 
+FilScan is a customized blockchain engine for Filecoin with searchable and analyzed data that allows all the actors to explore the valuable information in a fast and easy way. It could be accessible by standard web browsers and responsive on mobile from anywhere in the world.
+ 
+The project will include the following sections: 
+ 
+ 1. Dashboard: Real-time Blockchain summary statistics, Search Bar(Keywords: Best Block, Peer ID and Wallet Address), Navigation Bar, Top 10 Tipset and Top-10 Message.
```

Comment:
Can you clarify what you mean with the specific keywords for the search bar?

Reply:
Also how do you measure top 10 tipset and top 10 messages? Or is this just the most recent tipsets/messages?

Reply:
The specific keywords include the CID of block, the CID of message, the address of user, the address of miner, or peer id of node, maybe some keywords that the filscan user wanna search.
You got it! We will show the latest chain info on the main page. If someone need view more detail,  we provide subpage for that.